spring-cloud-stream/src/main/java/org/springframework/cloud/stream/annotation/Input.java

```diff
@@ -42,4 +44,8 @@
 
 	String value() default "";
 
+	String destination() default Sink.INPUT;
```

Comment:
Looks like `2.0.x` isn't rebased to `master`: https://github.com/spring-cloud/spring-cloud-stream/blob/master/spring-cloud-stream/src/main/java/org/springframework/cloud/stream/annotation/Input.java#L44

And here is no JavaDocs.

There is also no functionality around these new attributes.
Are you going to address it lately?
Why then do not introduce them there as well?

This PR doesn't bring value to the Framework, but may make some noise from the community:

> I try to use new attributes in my `@Input` annotation but they seems don't work. Am I doing anything wrong?

Does it make sense to you?

(Sorry if I sound as nasty)

Reply:
@artembilan Very important cue here: "Just adding the properties, will issue separate PR for internal change
in logic using them."